### ISSUE TITLE
[BUG] bootstrap + update now complains about unsupported architecture

### ISSUE THREAD

**Issue by cedwards** [BastilleBSD/bastille#245]
**[MANDATORY] Describe the bug [MANDATORY]**
```
root@01-host:~ # bastille bootstrap 11.4-RELEASE update
Unsupported architecture.
root@01-host:~ # bastille bootstrap 11.4-RELEASE
Bootstrapping FreeBSD distfiles...
```
**[MANDATORY] Bastille and FreeBSD version (paste ``bastille -v && freebsd-version -kru`` output)**
```
0.7.20200714
12.1-RELEASE-p8
```
**[MANDATORY] How did you install bastille? (port/pkg/git)**
git

**[optional] Steps to reproduce?**
`bastille bootstrap RELEASE update`

**[optional] Expected behavior**
release is bootstrapped and updates applied..

At first glance I assume this was introduced with @JRGTH patch to add support for i386 jails.

Please have a look and make sure we can support the existing `update` option as well as the i386 addition.

**Comment by JRGTH** [comment]
This should be addressed with the following PR #246 
Regards

**Comment by cedwards** [comment]
This looks resolved.
